packages/roosterjs-editor-core/lib/corePlugins/TypeInContainerPlugin.ts

```diff
@@ -46,10 +59,11 @@ export default class TypeInContainerPlugin implements EditorPlugin {
             let range = this.editor.getSelectionRange();
 
             if (
-                !range ||
-                this.editor.contains(
-                    findClosestElementAncestor(range.startContainer, null /* root */, '[style]')
-                )
+                !this.isRangeEmpty(range) &&
```

Comment:
You can move this check after line 63 so that we know range must not be null then no need to check again in line 35.

When enable strict mode, current code will fail the build because line 33 requires a range object but here it could benull